templates/docs/rhdm712-authoring-ha.adoc

```diff
@@ -92,29 +92,27 @@ https://docs.okd.io/latest/architecture/core_concepts/templates.html#parameters[
-|`AUTH_LDAP_ROLE_ATTRIBUTE_IS_DN` | `AUTH_LDAP_ROLE_ATTRIBUTE_IS_DN` | Whether or not the roleAttributeID contains the fully-qualified DN of a role object. If false, the role name is taken from the value of the roleNameAttributeId attribute of the context name. Certain directory schemas, such as Microsoft Active Directory, require this attribute to be set to true. | false | False
-|`AUTH_LDAP_REFERRAL_USER_ATTRIBUTE_ID_TO_CHECK` | `AUTH_LDAP_REFERRAL_USER_ATTRIBUTE_ID_TO_CHECK` | If you are not using referrals, you can ignore this option. When using referrals, this option denotes the attribute name which contains users defined for a certain role, for example member, if the role object is inside the referral. Users are checked against the content of this attribute name. If this option is not set, the check will always fail, so role objects cannot be stored in a referral tree. | -- | False
-|`AUTH_ROLE_MAPPER_ROLES_PROPERTIES` | `AUTH_ROLE_MAPPER_ROLES_PROPERTIES` | When present, the RoleMapping Login Module will be configured to use the provided file. This parameter defines the fully-qualified file path and name of a properties file or resource which maps roles to replacement roles. The format is original_role=role1,role2,role3 | -- | False
-|`AUTH_ROLE_MAPPER_REPLACE_ROLE` | `AUTH_ROLE_MAPPER_REPLACE_ROLE` | Whether to add to the current roles, or replace the current roles with the mapped ones. Replaces if set to true. | -- | False
+|`AUTH_LDAP_NEW_IDENTITY_ATTRIBUTES` | `AUTH_LDAP_NEW_IDENTITY_ATTRIBUTES` | Provide new identities for Ldap  identity mapping, the pattern to be used with this env is 'attribute_name=attribute_value;another_attribute_name=value' | sn=BlankSurname;cn=BlankCommonName | False
+|`AUTH_LDAP_REFERRAL_MODE` | `AUTH_LDAP_REFERRAL_MODE` | If LDAP referrals should be followed. Corresponds to REFERRAL ('java.naming.referral') environment property. Allowed values: 'ignore', 'follow', 'throw' | -- | False
+|`AUTH_ROLE_MAPPER_ROLES_PROPERTIES` | `AUTH_ROLE_MAPPER_ROLES_PROPERTIES` | When present, the RoleMapping will be configured to use the provided properties file or roles. This parameter defines the fully-qualified file path and name of a properties file or a set of roles with the following pattern 'role=role1;another-role=role2'. The format of every entry in the file is original_role=role1,role2,role3 | role=role1,role3,role4;role7=role,admin | False
+|`AUTH_LDAP_MAPPER_KEEP_MAPPED` | `AUTH_LDAP_MAPPER_KEEP_MAPPED` | When set to 'true' the mapped roles will retain all roles, that have defined mappings. Defaults to false. | -- | False
+|`AUTH_LDAP_MAPPER_KEEP_NON_MAPPED` | `AUTH_LDAP_MAPPER_KEEP_NON_MAPPED` | When set to 'true' the mapped roles will retain all roles, that have no defined mappings. Defaults to false. | -- | False
```

Comment:
If false, roles that are not mentioned in the mapping file are just deleted?

Reply:
if false, all roles will be kept.